iptables/iptables.go

```diff
@@ -451,6 +457,17 @@ func (ipt *IPTables) runWithOutput(args []string, stdout io.Writer) error {
 	return nil
 }
 
+// lookIptablesDefaultPath searches the iptables binaries at the default paths.
+func lookIptablesDefaultPath(command string) (string, error) {
+	for _, prefix := range IPTABLES_DEFAULT_PREFIXES {
+		path := prefix + "/" + command
+		if _, err := os.Stat(path); err == nil {
```

Comment:
This is probably good enough, but you could do a syscall.Access check on path to make sure it is executable.  Not sure if their is a different golang call for this.

Reply:
For this one, actually I've found a better solution. I found that exec.LookPath() actually accepts relative/absolution paths. If we pass a path to it, it will ignore the $PATH variable, just check if the given path points to a valid executable.